src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,34 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/add")
+public class CreateDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher(
+                "/WEB-INF/views/drivers/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        Driver driver = new Driver(req.getParameter("name"),
+                req.getParameter("licenseNumber"));
+        driverService.create(driver);
+        resp.sendRedirect("/index");
```

Comment:
Show her all drivers after creating
```suggestion
        resp.sendRedirect("/drivers");
```